go.mod

```diff
@@ -139,7 +139,7 @@ require (
 	github.com/ryanuber/go-glob v1.0.0
 	github.com/safchain/ethtool v0.4.1
 	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.30
-	github.com/siderolabs/crypto v0.4.4
+	github.com/siderolabs/crypto v0.5.0
```

Comment:
can we split up this PR into the piece which only bumps crypto and fixes ALPN (which we can backport to 1.8), and another PR which does the rest of updates?

Reply:
Will do.